tools/clang/test/CodeGenSPIRV/spirv.interpolation.hlsl

```diff
@@ -34,6 +34,35 @@ struct PSInput {
   nointerpolation      bool3 bool_d: BOOLD;
 };
 
-float4 main(PSInput input) : SV_Target {
+float4 main(                     PSInput input,
+                                 float   fp_a: FPA1,
```

Comment:
is there no decoration for `linear` keyword?

Reply:
Yes, that's the default.

Reply:
Thanks